dags/crash_aggregates.py

```diff
@@ -0,0 +1,23 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from operators.emr_spark_operator import EMRSparkOperator
+
+default_args = {
+    'owner': 'mdoglio@mozilla.com',
+    'depends_on_past': True,
+    'start_date': datetime(2016, 6, 27),
```

Comment:
I don't think so; that's the start date of the dag, i.e. the first time the dag should ever be scheduled.